src/v/kafka/server/handlers/fetch.cc

```diff
@@ -593,6 +629,13 @@ static ss::future<> fetch_topic_partitions(op_context& octx) {
       octx.request.data.max_wait_ms));
 }
 
+namespace testing {
```

Comment:
Note sure if we have active convention here, but I put this in the `testing::` namespace to indicate it is exposed for testing purposes only.

Reply:
(we benchmark this method specifically)